docs/usage.md

```diff
@@ -2,88 +2,88 @@
+    * [`--saveTrimmed`](#--savetrimmed)
+    * [`--saveUnaligned`](#--saveunaligned)
+    * [`--saveAlignedIntermediates`](#--savealignedintermediates)
+    * [`--gencode`](#--gencode)
+      * ["Type" of gene](#type-of-gene)
+      * [Transcript IDs in FASTA files](#transcript-ids-in-fasta-files)
+    * [`--skipBiotypeQC`](#--skipbiotypeqc)
+    * [`--additional_fasta`](#--additional_fasta)
+    * [`--skipAlignment`](#--skipalignment)
+    * [Compressed Reference File Input](#compressed-reference-file-input)
+      * [Create compressed (tar.gz) STAR indices](#create-compressed-targz-star-indices)
+      * [HISAT2 indices](#hisat2-indices)
+      * [Salmon index](#salmon-index)
+  * [Adapter Trimming](#adapter-trimming)
+    * [`--clip_r1 [int]`](#--clip_r1-int)
+    * [`--clip_r2 [int]`](#--clip_r2-int)
+    * [`--three_prime_clip_r1 [int]`](#--three_prime_clip_r1-int)
+    * [`--three_prime_clip_r2 [int]`](#--three_prime_clip_r2-int)
+    * [`--trim_nextseq [int]`](#--trim_nextseq-int)
+    * [`--skipTrimming`](#--skiptrimming)
+  * [UMI handling](#umi-handling)
+    * [`--with-umi`](#--with-umi)
+    * [`--umitools_extract_method [str]`](#--umitools_extract_method-str)
+    * [`--umitools_bc_pattern [str]`](#--umitools_bc_pattern-str)
+    * [`--umitools_extract_extra [str]`](#--umitools_extract_extra-str)
+    * [`--umitools_dedup_extra [str]`](#--umitools_dedup_extra-str)
+    * [`--save_umi_itermediates`](#--save_umi_itermediates)
+  * [Ribosomal RNA removal](#ribosomal-rna-removal)
+    * [`--removeRiboRNA`](#--removeriborna)
+    * [`--saveNonRiboRNAReads`](#--savenonribornareads)
+    * [`--rRNA_database_manifest`](#--rrna_database_manifest)
+  * [Library Prep Presets](#library-prep-presets)
+    * [`--pico`](#--pico)
+  * [Skipping QC steps](#skipping-qc-steps)
+  * [Job resources](#job-resources)
+    * [Automatic resubmission](#automatic-resubmission)
+    * [Custom resource requests](#custom-resource-requests-1)
+    * [Running in the background](#running-in-the-background)
+      * [Nextflow memory requirements](#nextflow-memory-requirements)
+  * [Other command line parameters](#other-command-line-parameters)
+    * [`--outdir`](#--outdir)
+    * [`--email`](#--email)
+    * [`--email_on_fail`](#--email_on_fail)
+    * [`--max_multiqc_email_size`](#--max_multiqc_email_size)
+    * [`--publish_dir_mode`](#--publish_dir_mode)
+    * [`-name`](#-name)
+    * [`-resume`](#-resume)
+    * [`-c`](#-c)
+    * [`--custom_config_version`](#--custom_config_version)
+    * [`--custom_config_base`](#--custom_config_base)
+    * [`--max_memory`](#--max_memory)
+    * [`--max_time`](#--max_time)
+    * [`--max_cpus`](#--max_cpus)
+    * [`--hisat_build_memory`](#--hisat_build_memory)
+    * [`--sampleLevel`](#--samplelevel)
+    * [`--percent_aln_skip`](#--percent_aln_skip)
+    * [`--plaintext_email`](#--plaintext_email)
+    * [`--monochrome_logs`](#--monochrome_logs)
+    * [`--multiqc_config`](#--multiqc_config)
```

Comment:
I think these params should be removed as they will be parsed from the json file, but can also do it in a separate PR together with the json docs

Reply:
Yeah I'm a bit lost on that part 😓